### ISSUE TITLE
nightly build crashing - Qt5Gui.dll & qwindows.dll

### ISSUE THREAD

**Issue by 2dvision** [opentoonz/opentoonz#418]
When I'm drawing, Opentoonz crash.
I think when i use "Full Screen Mode" and drawing, OT crash more.

Error message:

```
Problem signature:
  Problem Event Name:   APPCRASH
  Application Name: OpenToonz_1.0.exe
  Application Version:  0.0.0.0
  Application Timestamp:    5756a429
  Fault Module Name:    Qt5Gui.dll
  Fault Module Version: 5.6.0.0
  Fault Module Timestamp:   56d82c20
  Exception Code:   c0000005
  Exception Offset: 00000000001f0c92
  OS Version:   6.1.7601.2.1.0.256.1
  Locale ID:    1033
  Additional Information 1: 6300
  Additional Information 2: 63006faf623985b3921f5344d63b1b45
  Additional Information 3: 0bd4
  Additional Information 4: 0bd4f1f398a1d69bcdc036b018f1d1b3

Read our privacy statement online:
  http://go.microsoft.com/fwlink/?linkid=104288&clcid=0x0409

If the online privacy statement is not available, please read our privacy statement offline:
  C:\Windows\system32\en-US\erofflps.txt
```

Error message 2:

```
Problem signature:
  Problem Event Name:   APPCRASH
  Application Name: OpenToonz_1.0.exe
  Application Version:  0.0.0.0
  Application Timestamp:    572097a0
  Fault Module Name:    qwindows.dll
  Fault Module Version: 5.6.0.0
  Fault Module Timestamp:   56d82de6
  Exception Code:   c0000005
  Exception Offset: 00000000000149df
  OS Version:   6.1.7601.2.1.0.256.1
  Locale ID:    1033
  Additional Information 1: 0ced
  Additional Information 2: 0ced2723f0c04e932a795d9a6fa94cf7
  Additional Information 3: 96dd
  Additional Information 4: 96dd8c74d1d379f58a78ddfcdc6f7fb8

Read our privacy statement online:
  http://go.microsoft.com/fwlink/?linkid=104288&clcid=0x0409

If the online privacy statement is not available, please read our privacy statement offline:
  C:\Windows\system32\en-US\erofflps.txt
```

**Comment by kogaki** [comment]
You mean, does it start happening in nightly build? v1.0.2 works OK?

**Comment by 2dvision** [comment]
In v1.0.2 i give this error too. I usually use "Raster Level" level type, now i guess when i drawing and "Viewer window" zoom is 100% or less OT don't crash, but if zoom 800% OT crash.

**Comment by turtletooth** [comment]
Is this still an issue?

**Comment by 2dvision** [comment]
It seems this issue is solved in V1.1.2.

**Comment by turtletooth** [comment]
Thanks
